lua/wtf/gpt.lua

```diff
@@ -53,7 +53,21 @@ local function get_model_id()
 end
 
 local function get_api_key()
-  local api_key = config.options.openai_api_key
+  local api_key_cmd = config.options.openai_api_key_cmd -- Check for command in config
```

Comment:
Would you mind putting the comments above the lines like we do elsewhere? It keeps the line length down :-)